prow/plugins/trigger/push_test.go

```diff
@@ -151,61 +153,131 @@ func TestHandlePE(t *testing.T) {
 			},
 			jobsToRun: 1,
 		},
-	}
-	for _, tc := range testCases {
-		g := &fakegithub.FakeClient{}
-		fakeProwJobClient := fake.NewSimpleClientset()
-		c := Client{
-			GitHubClient:  g,
-			ProwJobClient: fakeProwJobClient.ProwV1().ProwJobs("prowjobs"),
-			Config:        &config.Config{ProwConfig: config.ProwConfig{ProwJobNamespace: "prowjobs"}},
-			Logger:        logrus.WithField("plugin", PluginName),
-		}
-		postsubmits := map[string][]config.Postsubmit{
-			"org/repo": {
-				{
-					JobBase: config.JobBase{
-						Name: "pass-butter",
+		{
+			name: "postsubmit job has PR reference",
```

Comment:
This needs test cases for push events that have multiple commits and multiple PRs in various states (merged and not merged)

Reply:
Thank you for the review. Fixed.